### ISSUE TITLE
pulsar-client-admin is missing deps in maven

### ISSUE THREAD

**Issue by piotrblasiak** [apache/incubator-pulsar#1409]
#### Expected behavior

Include the pulsar-client-admin artifact and be able to instantiate and use PulsarAdmin.

#### Actual behavior

There seems to be missing dependencies according to the exception:

```
java.lang.RuntimeException: java.lang.ClassNotFoundException: Provider org.apache.pulsar.admin.shade.org.glassfish.jersey.internal.RuntimeDelegateImpl could not be instantiated: java.lang.IllegalStateException: No generator was provided and there is no default generator registered
	at org.apache.pulsar.admin.shade.javax.ws.rs.ext.RuntimeDelegate.findDelegate(RuntimeDelegate.java:152)
	at org.apache.pulsar.admin.shade.javax.ws.rs.ext.RuntimeDelegate.getInstance(RuntimeDelegate.java:120)
```

#### Steps to reproduce

```
<dependency>
			<groupId>org.apache.pulsar</groupId>
			<artifactId>pulsar-client-admin</artifactId>
			<version>${pulsar.version}</version>
		</dependency>
```

#### System configuration
**Pulsar version**: 1.22.0-incubating

**Comment by piotrblasiak** [comment]
I was told to use pulsar-client-admin instead but that in turn gave me a deps problem:

`java.lang.NoSuchMethodError: com.fasterxml.jackson.jaxrs.cfg.EndpointConfigBase.<init>(Lcom/fasterxml/jackson/databind/cfg/MapperConfig;)V`

It seems that the whole pulsar project depends on jersey-media-json-jackson 2.9, which in turn depends on jackson-jaxrs-base 2.3.2 while the jackson version used in the pulsar project is actually much higher (2.9.3). Is this a bug or is it just because I use the pulsar-client-admin artifact?

**Comment by zubchenok** [comment]
Having the same issue. Please fix it.

**Comment by zubchenok** [comment]
Does not work pulsar-client-admin, but works with **pulsar-client-admin-original** version 1.22-incubating

Gradle workaround:
```
    compile 'com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:2.9.3'
    compile('org.apache.pulsar:pulsar-client-admin-original:1.22.0-incubating') {
        exclude group: 'com.fasterxml.jackson.jaxrs', module: 'jackson-jaxrs-json-provider'
    }
```

**Comment by zubchenok** [comment]
I also get a warning `Cannot find a default implementation of the HK2 ServiceLocatorGenerator` on app start

**Comment by sijie** [comment]
This issue of pulsar-client-admin  should be fixed by #1370 and it will be released in the upcoming 2.0.0 release. It should be coming out in this month.

will take a look into the pulsar-client-admin-original one. @zubchenok @piotrblasiak
